src/app/components/TradeProfit/index.tsx

```diff
@@ -1,50 +1,34 @@
-import React, { useState, useEffect, useMemo } from 'react';
+import React, { useState, useEffect } from 'react';
 import { bignumber } from 'mathjs';
 import { Tooltip } from '@blueprintjs/core';
 import { useTranslation } from 'react-i18next';
 import classNames from 'classnames';
 import { TradingPosition } from 'types/trading-position';
-import {
-  toNumberFormat,
-  weiToAssetNumberFormat,
-} from 'utils/display-text/format';
+import { toAssetNumberFormat } from 'utils/display-text/format';
 import { translations } from 'locales/i18n';
-import { assetByTokenAddress } from 'utils/blockchain/contract-helpers';
-import { TradingPairDictionary } from 'utils/dictionaries/trading-pair-dictionary';
-import { getTradingPositionPrice } from 'app/pages/MarginTradePage/utils/marginUtils';
 import { AssetRenderer } from '../AssetRenderer';
-import { LoanEvent } from 'app/pages/MarginTradePage/components/OpenPositionsTable/hooks/useMargin_getLoanEvents';
+import { Asset } from 'types/asset';
 
 interface ITradeProfitProps {
-  closedItem: LoanEvent;
-  openedItem: LoanEvent;
+  collateralAsset: Asset;
+  position: TradingPosition;
+  openPrice: string;
+  closePrice: string;
+  realizedPnL: string;
+  realizedPnLPercent: string;
 }
 
 export const TradeProfit: React.FC<ITradeProfitProps> = ({
-  closedItem,
-  openedItem,
+  collateralAsset,
+  position,
+  openPrice,
+  closePrice,
+  realizedPnL,
```

Comment:
Does this come from the graph? It should be calculated from the contract as in the old version. Only properties that used to come from the legacy backend should be replaced by graph values

Reply:
No, it doesn't. only the realizedPnL, and I already replaced it